sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1713,4 +1713,19 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
       assert(sql("show user functions").count() === 1L)
     }
   }
+
+  test("show columns - negative test") {
+    // When case sensitivity is true, the user supplied database name in table identifier
+    // should match the supplied database name in case sensitive way.
+    withSQLConf(SQLConf.CASE_SENSITIVE.key -> "true") {
+      val tabName = "showcolumn"
+      withTable(tabName) {
+        sql(s"CREATE TABLE $tabName(col1 int, col2 string) USING parquet ")
```

Comment:
Do we need to explicitly set the current database here? If the current database is not `default`, the test is not making sense anymore.